src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs

```diff
@@ -304,6 +317,18 @@ public class CommitmentOrchestrator : BaseCommitmentOrchestrator
             return data.Commitments.Where(m => m.GetStatus() == requestStatus);
         }
 
+        public async Task<IEnumerable<CommitmentListItem>> GetAllCommitmentsWithTheseStatuses(long providerId, RequestStatus[] requestStatus)
```

Comment:
You could use params here to make the calling code look a little better.

Reply:
done